scripts/install.ps1

```diff
@@ -1,8 +1,19 @@
+param ($proxyUrl)
 $p = New-Object System.Security.Principal.WindowsPrincipal([System.Security.Principal.WindowsIdentity]::GetCurrent())
 if (!$p.IsInRole([System.Security.Principal.WindowsBuiltInRole]::Administrator)) {
     throw 'This script requires admin privileges to run and the current Windows PowerShell session is not running as Administrator. Start Windows PowerShell by using the Run as Administrator option, and then try running the script again.'
 }
 [Net.ServicePointManager]::SecurityProtocol = 'tls12, tls';
-$version = (New-Object System.Net.WebClient).DownloadString("https://download.newrelic.com/install/newrelic-cli/currentVersion.txt").Trim();
-(New-Object System.Net.WebClient).DownloadFile("https://download.newrelic.com/install/newrelic-cli/${version}/NewRelicCLIInstaller.msi", "$env:TEMP\NewRelicCLIInstaller.msi");
+$WebClient = New-Object System.Net.WebClient
+if ($proxyUrl) {
```

Comment:
Is there a reason why we wouldn't use the same HTTPS_PROXY mechanism for windows?

Reply:
No specific reason other than thinking that it would be more flexible to pass it as a script parameter than to set the user environment, but also no reason to break from convention. Pushing up a change for this momentarily